d3-cli/tests/test_d3_build.py

```diff
@@ -57,7 +57,8 @@ def test_invalid_uri(caplog):
 
 def test_non_existent_parent_behaviour():
     """Test whether behaviours with non-existent parents raises an error"""
-    test_dir = Path(__file__).parent / "__fixtures__" / "non-existent-parent-behaviour"
+    test_dir = Path(__file__).parent / "__fixtures__" / \
+        "non-existent-parent-behaviour"
```

Comment:
It might be neater to use `()` instead of using `\`

As a side-note, [PEP 8 (Python style-guide) also recommends having the line-break before the operand](https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator), e.g.

```suggestion
    test_dir = (Path(__file__).parent
                / "__fixtures__"
                / "non-existent-parent-behaviour")
```